tests/container/cloudbuild.yaml

```diff
@@ -0,0 +1,85 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+options:
+    machineType: "N1_HIGHCPU_32"
+
+steps:
+  - name: "l.gcr.io/google/bazel"
+    args: 
```

Comment:
Can we setup these tests to execute on RBE and not need a N1_HIGHCPU_32 machine?

Reply:
Done